openwisp_utils/api/urls.py

```diff
@@ -0,0 +1,38 @@
+from django.conf.urls import url
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from openwisp_utils import settings as app_settings
+from rest_framework import permissions
+
+urlpatterns = []
+
+if app_settings.REST_SWAGGER:
+    schema_view = get_schema_view(
+        openapi.Info(
+            title='OpenWISP API',
+            default_version='v1',
+            description='OpenWISP REST API',
+            contact=openapi.Contact(email='openwisp@googlegroups.com'),
+            license=openapi.License(name="GPLv3"),
```

Comment:
I'm pretty sure users will need to modify this info since they tend to use openwisp as a base for their tailored solutions.

could you make all these arguments configurable with a dict?

Eg:

```python
API_INFO = getattr(settings, 'OPENWISP_API_INFO', {
    # ...
})

openapi.Info(**API_INFO)
```

Reply:
done